src/test/java/api/support/builders/InstanceBuilder.java

```diff
@@ -84,10 +84,15 @@ public Builder withInstanceTypeId(UUID instanceTypeId) {
   }
 
   public InstanceBuilder withContributor(String name, UUID typeId) {
+    return withContributor(name, typeId, null);
```

Comment:
Why is primary set to null rather than false?

Reply:
fixed, thanks